requirements.txt

```diff
@@ -10,3 +10,6 @@ dlipower
 cdrouter
 pyserial
 ipaddress
+pysnmp
+selenium
+pyasn1
```

Comment:
Can we add these in your own layer? We will make our automation scripts run pip install on all the overlays if the file exists.

boardfarm-cbn/requirements.txt